### DISCUSSION TITLE
Node/Layer management

### DISCUSSION THREAD

**Discussion by melMass** [nxt-dev/nxt_editor#230] [Ideas]
Hi,

It would be nice to be able to move nodes into another layer, or is it possible?
I also seem to randomly get errors while deleting nodes: They either grey out and muting/unmuting the layer the truly removes it. In other instances it clashes with the hover event (the target being null) and displays an error (I will try to track it down better).

Thanks

**Comment by melMass** [comment]
> It would be nice to be able to move nodes into another layer, or is it possible?

In fact Cut/Paste works very well for that

**Reply by ImLucasBrown** [reply]
You can use Find and Replace (`Ctrl+F`) to strip off the `pasted__` once you're done.
Another option is to right click a node and select "Localize Node", that will functionally copy it from it's layer to the target layer.
